### ISSUE TITLE
Error occurred while running npm scripts

### ISSUE THREAD

**Issue by Panepo** [prescottprue/generator-react-firebase#94]
**Do you want to request a *feature* or report a *bug*?**  
Bug

**What is the current behavior?**
The first error occurred while running scripts: `npm run lint`
```
> eslint .

Invalid `"trailingComma"` value. Expected "all", "es5" or "none", but received `"es6"`.
Error: Invalid `"trailingComma"` value. Expected "all", "es5" or "none", but received `"es6"`.
```
This error is not shown after modify the prettier config of `.eslintrc`
```
trailingComma: 'none',
```

The second error occurred while running scripts: `npm run test`
```
> cross-env NODE_ENV=test karma start build/karma.config


START:

ERROR in ./tests/routes/Home/components/HomePage.spec.js
Module not found: Error: Can't resolve 'routes/Home/components/Home' in 'D:\Github\Trentini\tests\routes\Home\components'
 @ ./tests/routes/Home/components/HomePage.spec.js 7:12-50
 @ ./tests \.(spec|test)\.(js|ts|tsx)$
 @ ./tests/test-bundler.js
PhantomJS 2.1.1 (Windows 8.0.0) WARN: 'Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.'
PhantomJS 2.1.1 (Windows 8.0.0) WARN: 'Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.'
PhantomJS 2.1.1 (Windows 8.0.0) ERROR
  Error: Cannot find module "routes/Home/components/Home"
  at webpack:///tests/routes/Home/components/HomePage.spec.js:2:0 <- tests/test-bundler.js:167280
PhantomJS 2.1.1 (Windows 8.0.0) ERROR
  Error: Cannot find module "routes/Home/components/Home"
  at webpack:///tests/routes/Home/components/HomePage.spec.js:2:0 <- tests/test-bundler.js:167280
```
This error is not shown after modify `.\tests\routes\Home\components\Home.spec.js`
```
import Home from 'routes/Home/components/HomePage'
```
The third error occurred while running scripts: `npm run test` again
```
> cross-env NODE_ENV=test karma start build/karma.config


START:
PhantomJS 2.1.1 (Windows 8.0.0) WARN: 'Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.'
PhantomJS 2.1.1 (Windows 8.0.0) WARN: 'Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.'
ERROR: 'Warning: Shallow renderer has been moved to react-test-renderer/shallow. Update references to remove this warning.'
ERROR: 'Warning: Shallow renderer has been moved to react-test-renderer/shallow. Update references to remove this warning.'
  (Component) LoadingSpinner
    √ Renders div
  (Layout) Core
    √ Should render as a <div>.
  (Account:Component) AccountForm
    × Renders div (skipped)
  (Route: Account Component) AccountPage
    × "before each" hook for "Renders div"
PhantomJS 2.1.1 (Windows 8.0.0) (Route: Account Component) AccountPage "before each" hook for "Renders div" FAILED
        Could not find "store" in either the context or props of "Connect(UserIsAuthenticated(withFirebase(withContext(getContext(withHandlers(Connect(branch(withHandlers(AccountPage)))))))))". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(UserIsAuthenticated(withFirebase(withContext(getContext(withHandlers(Connect(branch(withHandlers(AccountPage)))))))))".
        invariant@webpack:///node_modules/invariant/browser.js:45:0 <- tests/test-bundler.js:2542:16
        Connect(UserIsAuthenticated(withFirebase(withContext(getContext(withHandlers(Connect(branch(withHandlers(AccountPage)))))))))@webpack:///node_modules/react-redux/es/components/connectAdvanced.js:117:0 <- tests/test-bundler.js:64042:58
        webpack:///node_modules/react-test-renderer/lib/shallow/ReactCompositeComponent.js:292:0 <- tests/test-bundler.js:126591:31
        measureLifeCyclePerf@webpack:///node_modules/react-test-renderer/lib/shallow/ReactCompositeComponent.js:73:0 <- tests/test-bundler.js:126372:14
        _constructComponentWithoutOwner@webpack:///node_modules/react-test-renderer/lib/shallow/ReactCompositeComponent.js:291:0 <- tests/test-bundler.js:126590:36
        mountComponent@webpack:///node_modules/react-test-renderer/lib/shallow/ReactCompositeComponent.js:185:0 <- tests/test-bundler.js:126484:40
        mountComponent@webpack:///node_modules/react-test-renderer/lib/shallow/ReactReconciler.js:43:0 <- tests/test-bundler.js:34080:49
        _render@webpack:///node_modules/react-test-renderer/lib/shallow/ReactShallowRenderer.js:136:0 <- tests/test-bundler.js:126281:37
        _batchedRender@webpack:///node_modules/react-test-renderer/lib/shallow/ReactShallowRenderer.js:83:0 <- tests/test-bundler.js:126228:19
        batchedUpdates@webpack:///node_modules/react-test-renderer/lib/shallow/ReactDefaultBatchingStrategy.js:58:0 <- tests/test-bundler.js:128163:22
        batchedUpdates@webpack:///node_modules/react-test-renderer/lib/shallow/ReactUpdates.js:95:0 <- tests/test-bundler.js:34301:41
        render@webpack:///node_modules/react-test-renderer/lib/shallow/ReactShallowRenderer.js:110:0 <- tests/test-bundler.js:126255:32
        render@webpack:///node_modules/enzyme/build/react-compat.js:171:0 <- tests/test-bundler.js:9896:41
        webpack:///node_modules/enzyme/build/ShallowWrapper.js:128:0 <- tests/test-bundler.js:58518:32
        perform@webpack:///node_modules/react-test-renderer/lib/shallow/Transaction.js:141:0 <- tests/test-bundler.js:34716:24
        batchedUpdates@webpack:///node_modules/react-test-renderer/lib/shallow/ReactDefaultBatchingStrategy.js:60:0 <- tests/test-bundler.js:128165:33
        batchedUpdates@webpack:///node_modules/react-test-renderer/lib/shallow/ReactUpdates.js:95:0 <- tests/test-bundler.js:34301:41
        unstable_batchedUpdates@webpack:///node_modules/react-test-renderer/lib/shallow/ReactShallowRenderer.js:128:0 <- tests/test-bundler.js:126273:39
        performBatchedUpdates@webpack:///node_modules/enzyme/build/ShallowWrapper.js:103:0 <- tests/test-bundler.js:58493:44
        webpack:///node_modules/enzyme/build/ShallowWrapper.js:127:0 <- tests/test-bundler.js:58517:30
        withSetStateAllowed@webpack:///node_modules/enzyme/build/Utils.js:284:0 <- tests/test-bundler.js:16611:5
        ShallowWrapper@webpack:///node_modules/enzyme/build/ShallowWrapper.js:126:0 <- tests/test-bundler.js:58516:38
        shallow@webpack:///node_modules/enzyme/build/shallow.js:19:0 <- tests/test-bundler.js:130503:41
        webpack:///tests/routes/Account/components/AccountPage.spec.js:9:0 <- tests/test-bundler.js:164230:38
```

**What is the expected behavior?**
There should be no error?

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**
Win10 (Build 16299)
node 8.9.1
npm

**Comment by prescottprue** [comment]
The lint issue does not appear on some versions of node, but I was able to replicate under 6.11.5, so we will switch it back to `'none'`.

As for the test errors, seems like that was due to a recent component name change.

Something else to note:
Work has started to offer Jest as an option for test framework (available [on the `jest` branch](https://github.com/prescottprue/generator-react-firebase/tree/jest)) -- The current default is karma/mocha/chai.
